tests/migrator/rules/test_books.py

```diff
@@ -1064,6 +1063,77 @@ def test_publication_info(app):
                 <subfield code="g">123456</subfield>
             </datafield>
             """,
+            {
```

Comment:
you might want to add a test for the PROCEEDING given the typo?

Reply:
the test is already below